plugins/modules/rabbitmq_user.py

```diff
@@ -120,6 +131,18 @@
         read_priv: .*
         write_priv: .*
     state: present
+
+# name: Add user to server and assign some topic permissions on / vhost.
```

Comment:
```suggestion
# name: Add user to server and assign some topic permissions on / vhost.
```
@abompard if you like to improve the module doc I have an idea (for another PR). This is not a recommended way of examples. It should be
```
- name: Add user ..
```
if a sentence is too long, we can use
```
- name: |-
  Add user ...
  ...
```

Reply:
No need to fix it right now. Would be nice to fix in all the modules with a separate PR